src/controllers/movie.js

```diff
@@ -0,0 +1,72 @@
+const { Prisma } = require("@prisma/client");
+const prisma = require("../utils/prisma");
+
+const getAllMovies = async (req, res) => {
+  const movies = await prisma.movie.findMany ({
+    include: {
+        screenings: true,
+      },
+     });
+
+  return res.status(200).json({ movies });
```

Comment:
One little point, it's not a problem but you don't always need to use "return" before your response.

Return serves two main purposes. It sends something somewhere and it exits a function.

So for example if you had a function (A) that created a constant, then called another function (B) which found a value for that constant, you would use return to have function (B) send the value of that constant back to function (A).

If you have your res.status line inside an if statement and you wanted the function to end after the if statement had been triggered then return would cause it to do that. For example:

const getALlMovies... {
if (id === undefined) {
...
return res.status(404)... "ID not found"
}
res.status(200)... "movie"
}

If the id is undefined, return this response and exit. Without the return the 200 status would still run, but with the return the function exits at that point.

If it's the last thing to happen in the function, as it is here, then return isn't necessary as the function ends after calling this line anyway.

Not a problem at all to use it though! I'm just trying to find something to comment on as per the instructions!